src/data/nat/factorization/basic.lean

```diff
@@ -527,4 +528,29 @@ begin
     convert (factorization_disjoint_of_coprime hab) },
 end
 
+/-! ### Lemmas about factorizations of particular functions -/
+
+-- TODO: Port lemmas from `data/nat/multiplicity` to here, re-written in terms of `factorization`
+
+/-- Exactly `n / p` naturals in `[1, n]` are multiples of `p`. -/
+lemma card_multiples (n p : ℕ) : card ((finset.range n).filter (λ e, p ∣ e + 1)) = n / p :=
+begin
+  induction n with n hn, { simp },
+  simp [nat.succ_div, add_ite, add_zero, finset.range_succ, filter_insert, apply_ite card,
+    card_insert_of_not_mem, hn],
+end
```

Comment:
```suggestion
begin
  induction n with n hn, 
  { simp },
  { simp [nat.succ_div, add_ite, add_zero, finset.range_succ, filter_insert, apply_ite card,
    card_insert_of_not_mem, hn] },
end
```
but I'm not sure this is even a golf, did you just change the `rw`s to `simp`?

Reply:
It's a golf from 281 characters to 168 characters, by replacing `rw [nat.zero_div, range_zero, filter_empty, card_empty]` in the base case with `{ simp }`, and changing the inductive step from a `rw` to a `simp` which allows us to avoid spelling out some details explcitly.

Reply:
The style guide says that "closing a goal with a proof that fits entirely on a single line" is ok and doesn't need to go on a separate line, and gives `induction n, { simp }` as an example.